modules/storages/spec/lib/api/v3/storage_files/storage_upload_link_representer_spec.rb

```diff
@@ -66,28 +66,5 @@
         end
       end
     end
-
-    describe 'with finalize link' do
-      let(:upload_link) do
-        Storages::UploadLink.new(
-          "https://example.com/upload/#{token}",
-          finalize
-        )
-      end
-
-      describe 'to destination' do
-        it_behaves_like 'has an untitled link' do
-          let(:link) { 'destination' }
-          let(:href) { destination }
-        end
-      end
-
-      describe 'to finalize' do
-        it_behaves_like 'has an untitled link' do
-          let(:link) { 'finalize' }
-          let(:href) { finalize }
-        end
-      end
-    end
```

Comment:
why did you remove the specs for the finalize link?

Reply:
Because they get implemented in a preceding WP: https://community.openproject.org/projects/openproject/work_packages/44880/activity

I wanted to keep this clean of thinks that do not matter in this WP here.